### PR TITLE
[For later] Update downloads page to target 1.17.1

### PR DESCRIPTION
**Pull request by simpleauthority**
When the time comes, this PR updates the downloads page to point at 1.17. However, it is gonna conflict with #88 unless edited as I did these at the same time.

edit: o wait if #88 merges then i can edit it myself nvm

### CODE REVIEW

File: src/js/downloads.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 const downloads = {
-    "Paper-1.16": {
-        "title": "Paper 1.16.5",
+    "Paper-1.17": {
+        "title": "Paper 1.17.1",
         "api_endpoint": "paper",
-        "api_version": "1.16",
-        "jenkins": "Paper-1.16",
+        "api_version": "1.17",
+        "jenkins": "Paper-1.17",
```
Comment (kashike) [inline src/js/downloads.js:6]: This should be able to be just removed as far as I know - unused field.

**Comment by simpleauthority** [comment]
superseded by #92

**Comment by MiniDigger** [comment]
sorry again 🙈

**Comment by simpleauthority** [comment]
Lol not a problem, as long as it got done ;)

**Pull request closed without merging**